python/ray/includes/unique_ids.pxi

```diff
@@ -42,14 +43,18 @@ cdef extern from "ray/constants.h" nogil:
 
 
 cdef class UniqueID:
-    cdef CUniqueID data
+    cdef CUniqueID *data
```

Comment:
Do we really want to allocate this on the heap? Seems like for such a small object it would be better to do it inline, can that be made to work?

Reply:
Yes, I removed all the heap allocations.